smart-contracts/zos.rinkeby.json

```diff
@@ -134,7 +167,7 @@
       {
         "address": "0x3c6e461341adbf7c0947085e86b4a6f35ff2f801",
         "version": "0.1.0",
-        "implementation": "0x7633dd082406861c384309c67576a4260c5775e0"
+        "implementation": "0x14bb3586ce2946e71b95fe00fc73dd30ed830863"
```

Comment:
👍 right @nfurfaro ?

Reply:
@julien51 Sorry I missed this! Yep, the implementation address should have changed, but not the proxy address.